### ISSUE TITLE
`ask diff` says both `skill.json` and `models/en-US.json` differ from the latest version

### ISSUE THREAD

**Issue by SijiaDavis** [alexa/alexa-skills-kit-sdk-for-nodejs#510]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

I was running into `[Error]: The local stored [skill] eTag does not match the one on the server side.` when attempting to deploy. While checking `ask diff`, it said both `skill.json` and `models/en-US.json` differ from the latest version, but it failed to point out the difference:

```
Your skill.json differs from the latest version.

Your models/en-US.json differs from the latest version.

Your Lambda function arn:aws:lambda:xxx is up to date.

No in-skill product for the current skill.

=========================== DIFF RESULT ===========================
===================================================================
--- local skill.json    
+++ remote skill.json   

===================================================================
--- local models/en-US.json     
+++ remote models/en-US.json    
```

## Expected Behavior
`ask diff` should not say the files differ from the latest version if there's no diff.

## Your Environment

* ASK SDK for Node.js used: 1.6.2
* Operating System and version: macOS Mojave version 10.14.3

## Node.js and NPM Info
* Node.js version used for development: 8.9.0
* NPM version used for development: 5.5.1

**Comment by RonWang** [comment]
Hey @SijiaDavis ,

Thank you for the post and I think this is a very valid question. 

Our consideration for "diff" is: we will tell users about version matching status (eTag match or not), and further, the content differences. 
The first message, e.g. "Your skill.json differs from the latest version", is to tell you that the version is different so be careful if you deploy. The eTag mismatch will need you to force the deploy to acknowledge the situation. Well the empty difference indicates that you don't have a difference in contents, but only on eTag.

Is this experience acceptable to you? Let me know if you have any idea of improvement.

Thanks,
Ron

**Comment by SijiaDavis** [comment]
@RonWang 
I appreciate the prompt response, yes that makes sense, but I wonder if it's possible to explicitly state that the difference lays in the version, in terms of the content.

**Comment by RonWang** [comment]
Hey @SijiaDavis 

We just released ask-cli@v1.6.5 where the diff command has been enhanced to display the diff result by version diff and content diff. CLI will display content diff only when the diff contains a meaningful comparison file for each line. Please check it out and let us know if that solves the confusion that you met.

Best,
Ron

**Comment by RonWang** [comment]
Close this issue. Please feel free to re-open if you still have question!
